README.md

```diff
@@ -32,6 +32,7 @@ settings before running these commands.:
  - `dbcUpload`: Uploads your Library to Databricks Cloud. Deletes the older version.
  - `dbcAttach`: Attaches your Library to the specified clusters.
  - `dbcRestartClusters`: Restarts the specified clusters.
+ - `dbcExecuteCommand`: Runs a command on a specified DBC Cluster
```

Comment:
Could you add this as a separate example. The information above is all related to `dbcDeploy`. Show how to use `dbcExecutionLanguage`, `dbcCommandFile`, and `dbcExecuteCommand` should be used. For example running a python script